components/PrimarySourceSetsComponents/SingleSet/SourceSetInfo/index.js

```diff
@@ -172,6 +175,25 @@ class SourceSetInfo extends React.Component {
                     title={set.name.replace(/\*/g, "")}
                   />
                 </div>
+                <div className={classNames.tools}>
+                  <div className={classNames.toolLinkAndIcon}>
+                    <img
+                      src={googleClassroom}
+                      alt=""
```

Comment:
Should this icon have alt text?

Reply:
no because it is decorative. empty alt lets screen readers ignore it.

Reply:
It turns out you shouldn't have alt text when there's "real" text right next to it that says the same thing.